tasks/systemctl_show.json

```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Get systemctl show output of a unit",
+  "input_method": "stdin",
+  "parameters": {
+    "bin_path": {
+      "description": "Path to systemctl binary",
+      "type": "Optional[String[1]]",
+      "default": "/bin/systemctl"
```

Comment:
I think on most distros `/bin` is a symlink to `/usr/bin` and `/bin/`  is supposed to go away? Should we use `/usr/bin` directly?
```suggestion
      "default": "/usr/bin/systemctl"
```

Reply:
updated this just now :)